lib/services/execution_iframe.dart

```diff
@@ -133,7 +133,8 @@ window.onerror = function(message, url, lineNumber, colno, error) {
     if (modulesBaseUrl != null) {
       requireConfig = '''
 require.config({
-  "baseUrl": "$modulesBaseUrl"
+  "baseUrl": "$modulesBaseUrl",
+  waitSeconds: 60
```

Comment:
I'm guessing this content is inline JSON, so shouldn't `waitSeconds` be `"waitSeconds"`?

Reply:
Yup, it is; I'll update this.

The default timeout for require.js is 7 seconds - I was hitting that when using a hotspot connection via my phone.